cluster/gce/manifests/kube-apiserver.manifest

```diff
@@ -25,10 +25,12 @@
       }
     },
     "command": [
-                 "/bin/sh",
-                 "-c",
-                 "exec /usr/local/bin/kube-apiserver {{params}} --allow-privileged={{pillar['allow_privileged']}} 1>>/var/log/kube-apiserver.log 2>&1"
-               ],
+    	"/usr/local/bin/kube-apiserver"
+    ],
+	"args": [
+  		"--allow-privileged={{pillar['allow_privileged']}}",
+  		{{params}}
+	],
```

Comment:
nit: fix indentation. Looks like a tabs vs. spaces thing. Same in other manifests.